Makefile

```diff
@@ -1,10 +1,72 @@
-bin: deps
-	go install ./...
+SHELL := /bin/bash
```

Comment:
we should probably use `/bin/sh` as `/bin/bash` is not available in all systems.